scripts/buildPluginReadme.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Updates the FaustWP plugin's readme.txt changelog with the
+ * latest 3 releases found in the plugin's CHANGELOG.md file.
+ */
+
+const fs = require("fs");
```

Comment:
Since Node v11 you can get promisified versions of `readFile` and `writeFile` so you can use them instead of the `util.promisify` and it becomes a little bit neater:

https://nodejs.org/api/fs.html#fspromisesreadfilepath-options

```
const fs = require('fs/promises');
const path = require("path");

const readFile = (filename) => fs.readFile(filename, {encoding: "utf8"});
const writeFile = fs.writeFile;
```

Reply:
Addressed in #704